src/ai/backend/manager/models/vfolder.py

```diff
@@ -1627,7 +1627,7 @@ async def batch_load_by_id(
                 query,
                 cls,
                 ids,
-                lambda row: row["user"],
```

Comment:
Since user_id is provided as a separate argument, the id here should be the vfolder's id.